README.md

```diff
@@ -100,12 +100,20 @@ $ ./bin/phing setup-behat
 In order to run JavaScript in your Behat tests, you must launch a PhantomJS
 instance before. Use the `--debug` parameter to get more information. Please
 be sure that the webdriver's port you specify corresponds to the one in your
-Behat configuration (`wd_host: "http://localhost:8643/wd/hub"`).
+Behat configuration (`behat.wd_host.url: "http://localhost:8643/wd/hub"`).
```

Comment:
It's in the "build.properties.dist" with the sentence "behat.wd_host.url = http://localhost:8643/wd/hub". (and not in the Behat configuration - behat.xxx.yml.dist )
Maybe, the last sentence is a little confusing.